collectors/python.d.plugin/edgex/edgex.chart.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
+# Description: EdgeX Platform netdata python.d module
+# Author: Odysseas Lamtzidis (OdysLam)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Module Version: 0.2
+
+import json
+import threading
+
+from collections import namedtuple
+
+try:
+    from Queue import Queue
+except ImportError:
+    from queue import Queue
+
+from bases.FrameworkServices.UrlService import UrlService
+
+METHODS = namedtuple('METHODS', ['get_data', 'url', 'run'])
+
+ORDER = [
+    'events_throughput',
+    'events_count',
+    'devices_number',
+    'memory_alloc',
+    'memory_malloc',
+    'memory_frees',
+    'memory_liveObjects'
+]
+
+# default module values (can be overridden per job in `config`)
+update_every = 5
+
+CHARTS = {
+    'events_throughput':{
+        'options': [None, 'Events Throughput', 'events/s',
+                    'events throughput','edgex.events_readings_incr', 'line'],
+        'lines': [
+            ['events', None, 'incremental'],
+            ['readings', None, 'incremental']
+        ]
+    },
+    'events_count':{
+        'options': [None, 'Absolute Count', 'events', 'events count', 'edgex.events_readings_abs',
+                    'line'],
+        'lines': [
+            ['readings', None, 'absolute'],
+            ['events', None, 'absolute']
+        ]
+    },
+    'devices_number':{
+        'options': [None, 'devices count', 'devices', 'devices count', 'edgex.devices_number',
+                    'line'],
+        'lines': [
+            ['registered_devices', None, 'absolute'],
+        ]
+    },
+    'memory_alloc':{
+        'options': [None, 'Alloc',
+                    'bytes', 'memory metrics', 'edgex.memory_alloc', 'line'],
+        'lines': [
+            ['core_data_alloc', 'core_data', 'absolute'],
+            ['core_metadata_alloc', 'core_metadata', 'absolute'],
+            ['core_command_alloc', 'core_command', 'absolute'],
+            ['support_logging_alloc', 'support_logging', 'absolute']
+        ]
+    },
+    'memory_malloc':{
+        'options': [None, 'Mallocs', 'heap objects',
+                    'memory metrics','edgex.memory_malloc', 'line'],
+        'lines': [
+            ['core_data_malloc', 'core_data', 'absolute'],
+            ['core_metadata_malloc', 'core_metadata', 'absolute'],
+            ['core_command_malloc', 'core_command', 'absolute'],
+            ['support_logging_malloc', 'support_logging', 'absolute']
+        ]
```

Comment:
> // Mallocs is the cumulative count of heap objects allocated.
// The number of live objects is Mallocs - Frees.
Mallocs uint64

It seems it is not useful to have `cumulative count of heap objects allocated` as 'absolute' value.